lab5/lab5.sql

```diff
@@ -0,0 +1,98 @@
+-- 1. Create restaurant food item table
+CREATE TABLE food_items (
+    id INTEGER AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    description VARCHAR(255),
+    price FLOAT
+);
+
+-- 2. Create restaurant order statuses table
+CREATE TABLE orders (
+    id INTEGER PRIMARY KEY,
+    customer_name VARCHAR(255),
+    -- using String for the Date so that I can do MySQL command #12 with the LIKE keyword
+    created VARCHAR(255)
```

Comment:
should be using datetime field because you can do date comparison better than using like.